charms/worker/k8s/terraform/README.md

```diff
@@ -0,0 +1,61 @@
+# Terraform module for k8s
```

Comment:
I think there are some ways that it could be done, like you could make a generic charm module and then point them both at it. But id like to avoid that at the moment since it isn't in line with the Terraform charms spec. I could see a world where there is a generic module that has the base code, and then you adjust just the outputs and pass them forward.

Reply:
Thank you @asbalderson! That sounds reasonable, let's stay as close to the charm spec as possible and see how these things develop from there on.